MapboxNavigation/DefaultStyle.swift

```diff
@@ -13,7 +13,7 @@ extension UIColor {
 extension UIColor {
     // General styling
     fileprivate class var defaultTint: UIColor { get { return #colorLiteral(red: 0.1411764771, green: 0.3960784376, blue: 0.5647059083, alpha: 1) } }
-    fileprivate class var defaultTintStroke: UIColor { get { return #colorLiteral(red: 0.05882352963, green: 0.180392161, blue: 0.2470588237, alpha: 1) } }
+    fileprivate class var defaultTintStroke: UIColor { get { return #colorLiteral(red:0.18, green:0.49, blue:0.78, alpha:1.0) } }
```

Comment:
@1ec5 here is how we're looking now: 
![simulator screen shot jul 14 2017 8 28 37 am](https://user-images.githubusercontent.com/1058624/28218806-87f2c4e2-686e-11e7-8a80-beb5268c8533.png)
![simulator screen shot jul 14 2017 8 28 48 am](https://user-images.githubusercontent.com/1058624/28218807-87f3bbae-686e-11e7-8d72-577bc873f2b3.png)